Gemfile

```diff
@@ -38,5 +38,9 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
+group :test do
+  gem 'rspec-rails'
```

Comment:
add version to lock the `rspec-rails` gem

Reply:
When appropriate to implement testing I add it again with version